src/client/debug/debug-move.js

```diff
@@ -37,6 +37,12 @@ export class DebugMove extends React.Component {
       this.props.fn.apply(this, argArray);
     } catch (e) {
       error = '' + e;
+      if (
+        process.env.NODE_ENV === 'development' ||
+        process.env.NODE_ENV === 'test'
+      ) {
+        console.error(e);
```

Comment:
Use the logging API we have at https://github.com/google/boardgame.io/blob/master/src/core/logger.js instead.

Reply:
The exported functions in logger.js only print the error message which is not what I wish for. Should I add another function, e.g. `errorWithStack`?

Reply:
You can change logger.js to wrap console.info, warn and error respectively.